bitnami/kube-prometheus/templates/alertmanager/networkpolicy.yaml

```diff
@@ -0,0 +1,63 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+# Based on upstream configuration: https://github.com/prometheus-operator/kube-prometheus/commit/968cb8fd22a1e21e63e8702bc1c0a0a4547e6d15
+{{- if and .Values.alertmanager.enabled .Values.alertmanager.networkPolicy.enabled }}
+kind: NetworkPolicy
+apiVersion: {{ include "common.capabilities.networkPolicy.apiVersion" . }}
+metadata:
+  name: {{ template "kube-prometheus.alertmanager.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "kube-prometheus.alertmanager.labels" . | nindent 4 }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  podSelector:
+    matchLabels: {{- include "kube-prometheus.alertmanager.matchLabels" . | nindent 6 }}
+  policyTypes:
+    - Ingress
+    - Egress
+  {{- if .Values.alertmanager.networkPolicy.allowExternalEgress }}
+  egress:
+    - {}
+  {{- else }}
+  egress:
+    {{- if .Values.alertmanager.networkPolicy.extraEgress }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.alertmanager.networkPolicy.extraEgress "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- end }}
+  ingress:
+    {{/* TODO: Allow configuring alertmanager container ports */}}
+    - ports:
+        - port: 8080
+        - port: 9093
+        - port: 9094
```

Comment:
Shouldn't these values be set based on `.Values.alertmanager.containerPorts.*`?

Reply:
They are hardcoded in the operator. Comment updated